src/transformers/Twitter.js

```diff
@@ -1,23 +1,35 @@
 import { URL } from 'url';
-
 import { fetchOEmbedData } from './utils';
 
+export const name = 'twitter';
+
 export const shouldTransform = url => {
   const { host, pathname } = new URL(url);
-
   return (
     ['twitter.com', 'www.twitter.com'].includes(host) &&
     pathname.includes('/status/')
   );
 };
 
-export const getHTML = url =>
-  fetchOEmbedData(
-    `https://publish.twitter.com/oembed?url=${url}&dnt=true&omit_script=true`
-  ).then(({ html }) =>
+export const buildUrl = (url, options = {}) => {
+  // https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/get-statuses-oembed
+  const urlObj = new URL(`https://publish.twitter.com/oembed`);
+  urlObj.search = new URLSearchParams({
+    url,
+    dnt: true,
+    omit_script: true,
+    ...options.params,
```

Comment:
the babel/no-camelcase errors are coming from here (same with Instagram) if you passs an option that is not camelcase (like "hide_media").

How should this be handled?